### PR TITLE
Add some missing error responses

### PR DESCRIPTION
**Pull request by samselikoff**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -353,6 +353,18 @@ paths:
...
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorData'
+        '504':
+          description: 'Timeout'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorData'
```
Comment (samselikoff) [inline openapi.yaml:367]: Hassan mentioned GET to `/files/{id}/content` can return a 500 but didn't specify which code – I guessed it's a 504 Timeout here but let me know if it should be different!

**Review by Nutlope — APPROVED** [review]

**Comment by Nutlope** [comment]
@samselikoff Turns out I was wrong – the API only returns 200s and 500s today and doesn't return 404s or 504s. Can we update the PR to reflect this please?

**Comment by samselikoff** [comment]
How does that look?

**Comment by Nutlope** [comment]
LGTM thanks!

**Pull request merged**